crates/router/src/core/payments/operations/payment_create.rs

```diff
@@ -1038,6 +1037,7 @@ impl PaymentCreate {
             session_expiry: Some(session_expiry),
             request_external_three_ds_authentication: request
                 .request_external_three_ds_authentication,
+            frm_metadata: request.frm_metadata.clone(),
```

Comment:
do we validate the value here? or any serde_json::value can be put in the column?

Reply:
No we don't validate here, validation is there at connector level. Signifyd & Riskified expects few fields from frm_metadata.